source/codegen/metadata/nidmm/enums.py

```diff
@@ -68,48 +44,30 @@
     'ApertureTimeUnits': {
         'values': [
             {
-                'documentation': {
-                    'description': 'Seconds'
-                },
                 'name': 'NIDMM_VAL_SECONDS',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': 'Powerline Cycles'
-                },
-                'name': 'NIDMM_VAL_POWER_LINE_CYCLES',
+               'name': 'NIDMM_VAL_POWER_LINE_CYCLES',
```

Comment:
The spacing changed for 'name'. Make it line up with 'value' beneath.

Reply:
fixed